frontend/tests/cypress/component/orgFacilities/ManageFacility.cy.js

```diff
@@ -146,14 +146,15 @@ describe('<ManageFacility />', () => {
     cy.get('@routerPush').should('have.been.calledWith', { name: 'Report Change' });
   });
 
-  it('should not render any tabs without any permissions', () => {
```

Comment:
issue: I think we need to leave the existing test in because we will need it once the permission is in. You can .skip it for now and add a TODO.